tools/travis_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+from common_py.system.executor import Executor as ex
+
+TRAVIS_BUILD_PATH = os.environ['TRAVIS_BUILD_DIR']
+
+DOCKER_IMAGE_NAME = 'iotjs/js_remote_test:0.2'
+DOCKER_NAME = 'jsremote_docker'
+DOCKER_ROOT_PATH = '/root'
+
+# The path to js-remote-test in Docker.
+DOCKER_JSREMOTE_PATH = DOCKER_ROOT_PATH + '/js-remote-test/'
+
+# Commonly used commands and arguments.
+BASE_COMMAND = ['python', '-u', DOCKER_JSREMOTE_PATH + 'driver.py']
+RELEASE_ARG = ['--buildtype', 'release']
+DEBUG_ARG = ['--buildtype', 'debug']
+COMMON_ARGS = ['--no-flash', '--no-test', '--no-memstat']
+
+DEVICES = ['rpi2', 'artik530', 'artik053', 'stm32f4dis']
+
+def parse_option():
+    parser = argparse.ArgumentParser(
+         description='JS-remote-test Travis script.')
+
+    parser.add_argument('--check-signoff', action='store_true')
+    parser.add_argument('--device', choices=DEVICES, action='append')
+    parser.add_argument('--app', choices=['iotjs', 'jerryscript'], action='append')
+
+    option = parser.parse_args(sys.argv[1:])
```

Comment:
Isn't enough just a `parser.parse_args()`? Do you need to skip the first argument?